u-mode-api/src/lib.rs

```diff
@@ -50,18 +50,21 @@
 pub enum Error {
```

Comment:
Nit: Unrelated, but typo in commit message.

Reply:
Fixed.